Common/utils.ts

```diff
@@ -235,4 +235,8 @@ export function combine(object1: PlayerStatModel, object2: PlayerStatModel): Pla
 
 export function sigmoid(z: number, k: number) {
   return 1 / (1 + Math.exp(-z/k));
+}
+
+export function playerStatsHasData(playerStats: PlayerStatModel[]): boolean {
```

Comment:
nit?: could this be replaced with just !!playerStats.length

I have no clue what this meant when I was reading.

Reply:
Changed name + function